jax_unirep/evotuning.py

```diff
@@ -265,10 +265,19 @@ def objective(
         "high": len(sequences) * 3,
         "q": 1,
     }
+
+    learning_rate_kwargs = {
```

Comment:
```suggestion
    # Default settings for learning_rate_kwargs
    learning_rate_kwargs = {
```

Could you add an analogous comment on top of the `n_epochs_kwargs` line?